content/posts/0.23-release.md

```diff
@@ -0,0 +1,153 @@
+native execution, and we see comparable improvements with the WebAssembly
+runtime on our benchmark suite.
+
+## Load time and size optimizations
+
+Reducing the load time and size of a minimal web page using Pyodide has been a
+continuous effort. Several techniques have been implemented aiming to reduce
+download size,
+ - tests and some standard library modules are unvendored when Pyodide is
+   initialized with `loadPyodide({..., fullStdLib=False})`, the default
+   currently, and need to be explicitly loaded. In this release we additionally
+   unvendored `_pydecimal` and `pydoc_data` modules.
+ - Pyodide no longer uses Emscripten preload plugin, and `pyodide.asm.data`
+   file is removed, in favor of `python_stdlib.zip`. This would make it easier
+   to adjust the contents of this file if needed, particularly that
+   `loadPyodide` can now load it from a custom location using the `stdLibURL`
+   parameter.
+ - Python standard libraries are now vendored in a zipfile:
+   /lib/python{version}.zip in the in-browser MEMFS file system. If you need to
+   access the standard library source code, you need to unpack the zip file.
+
+To some extent, however, these optimizations have been offset by the
+growth of the Python standard library, and the enabling additional
+functionality over time. 
+
+As can be seen in the following graph, there are size increases for each major
+Python version, with an overall, slowly decreasing trend for the download size,
+
+<img src="download_size_evolution.webp" alt="Download size evolution for the Python REPL" />
+
+In this release we have also added a py-compiled build for the Python standard
+library and packages. This leads to faster initialization times, however
+because the size of .pyc files increased in Python 3.11
+([cpython#99554](https://github.com/python/cpython/issues/99554)) the overall
+download size is larger.
+
+--------------------------------------------
+|        | full | py-compiled |
+|--------|------|-------------|
+| Download size of Python + sdlib (Brotli compressed) |  5.3 MB |      6.9 MB    | 
+| Load + initialization time (Firefox) |  1.3 s |      0.87 s    |
+| Load + initialization time (Chrome)  |  0.85 s |      0.46 s    |
+
+Similarly load time improvements x1.2 - x2.0 were observed for larger packages
+such as pandas, sympy or scikit-learn
+([pyodide#3269](https://github.com/pyodide/pyodide/issues/3269))
+
+You can use to py-compiled builds by either setting,
+```
+https://cdn.jsdelivr.net/pyodide/v0.23.0/pyc/
+```
+as the `indexURL` or by appending `?build=pyc` to [the REPL
+URL](https://pyodide.org/en/stable/console.html?build=pyc). Pyodide CLI now
+also has the command `pyodide py-compile` to py-compile Python wheels to for a
+given Python version.
+
+A known limitation of py-compiled builds is that the error traceback no longer
+contain code snippets, only the line numbers.
+
+## Javascript API and Foreign Function Interface
+
```

Comment:
There's a fun story about Python multiple inheritance and my `tp_mro` hack here:
https://github.com/pyodide/pyodide/blob/main/src/core/jsproxy.c#L3893-L3917
The FFI is getting really polished at this point.

I think we should add better cross-language typing to our roadmap. This is the biggest area of the ffi that still needs work IMO.

Reply:
@hoodmane I added some things for this section, but not overly inspired. That is an interesting story, but I'm not sure how to integrate it into the release notes )